erizo_controller/erizoClient/src/ErizoConnectionManager.js

```diff
@@ -36,6 +36,7 @@ class ErizoConnection extends EventEmitterConst {
     this.sessionId = ErizoSessionId;
     this.connectionId = spec.connectionId;
     this.qualityLevel = QUALITY_LEVEL_GOOD;
+    this.reachedConnectivity = false;
```

Comment:
I'm wondering if `reachedConnectivity` is the best way to call this. I don't know... `wasAbleToConnect`, `wasPreviouslyConnected` seem clearer to me but it's your call :)